test/spec/tokenManager.js

```diff
@@ -0,0 +1,189 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
+  var tokens = require('../util/tokens');
+  var util = require('../util/util');
+
+  function setupSync(options) {
+    options = options || {};
+    return new OktaAuth({
+      url: 'https://auth-js-test.okta.com',
+      tokenManager: {
+        storage: options.type
+      }
+    });
+  }
+
+  beforeEach(function() {
+    localStorage.clear();
+    sessionStorage.clear();
+  });
+
+  describe('general tokenManager', function() {
```

Comment:
I think it'd be nicer to wrap this whole block in a describe('TokenManager') and then the individual describes could be describe('localStorage'), describe('sessionStorage'), etc.

Nicer in that when running in the browser it's easier to just test all TokenManager items at once, etc.